docs/core/porting/libraries.md

```diff
@@ -22,11 +22,11 @@ Porting is a task that may take time, especially if you have a large codebase.
 
 This article assumes you are using Visual Studio 2015 or later on Windows.  The bits required for building .NET Core code are not available on previous versions of Visual Studio.
 
-This article also assumes that you have understood the [overview of the porting process](overview.md) and that you have resolved any issues with [third-party dependencies](third-party-deps.md).
+This article also assumes that you have understood the [overview of the porting process](index.md) and that you have resolved any issues with [third-party dependencies](third-party-deps.md).
```

Comment:
I prefer present tense too. Didn't look at the overall sentence when I fixed the link. I'll fix that.